Templates/HspiSample.Dev/HspiSample.Dev.vbproj

```diff
@@ -413,14 +413,14 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="HomeSeerAPI, Version=1.0.2.1, Culture=neutral, processorArchitecture=x86">
-      <HintPath>..\..\packages\Homeseer.1.0.0\lib\net45\HomeSeerAPI.dll</HintPath>
+    <Reference Include="HomeSeerAPI, Version=1.0.2.1, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\homeseer.1.1.0\lib\net45\HomeSeerAPI.dll</HintPath>
```

Comment:
These are in the net45 directory but the target framework is net461. Is this right?